source/documentation/runbooks/internal/how-to-be-support.md.erb

```diff
@@ -33,26 +33,44 @@ Monitor this channel and respond to queries and requests as they come in. If you
 
 There are a number of email groups you should monitor for customer queries and requests. They are:
 
+TODO: Add links to the following
 - Ops Eng Google Group
 - Domains Google Group
 - Certificates Google Group
 - Certificate Alerts Google Group
 
 If a request comes in on the day you're on support, action it, unless the email is a follow on from a previous request.
+If the email is a follow on from a previous request, add it to the existing ticket.
```

Comment:
what is the existing ticket?

Reply:
This was a leftover sentence that needs to be removed. Good catch.